### DISCUSSION TITLE
openwrt 直接下载的linux_musl_amd64_alist文件 。运行后alist admin 复制密码 web登录提示密码错误！！！

### DISCUSSION THREAD

**Discussion by lj267lj267** [AlistGo/alist#2799] [Invalid question]
openwrt 直接下载的linux_musl_amd64_alist文件 。运行后alist admin 复制密码 web登录提示密码错误！！！使用alist serve 和 alist start 后，复制密码web 登录都提示登录密码错误。难道是有什么文档没看过？望帮忙提示一下怎么办

**Comment by lj267lj267** [comment]
这难道不是一个问题吗？

**Comment by lj267lj267** [comment]
终于知道原因了。先前运行时生成了data文件夹，初始密码在这个文件夹的数据库中保存。第二次运行时，如果用alist admin查看密码，会得到与data文件夹数据库中保存的不同的密码。作者认为这不是一个bug还是有点轻率。解决方法是删除掉所有先前运行alist产生的data文件夹就正常了。我是删除/根目录下的data文件夹好的。望能给后来者一个提示！！！

**Reply by xinxinenjoy** [reply]
这个的主要问题是：开机启动和手动运行，所调用的CMD不是一个路径
注意不要弄混就行了
